src/index.js

```diff
@@ -2,9 +2,10 @@
 var ga = require('./ga');
 
 
-function analytics(state) {
+function analytics(state, anonymizeIp) {
```

Comment:
I never used any other options for ga, but this looks useful, but I think other options might be useful too, so I think it would be better to an optional `options` object.

Can you change this, if not I want to take a look at it.